Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -140,14 +140,19 @@ - (void)verifyPhoneNumber:(NSString *)phoneNumber
         completion(nil, error);
         return;
       }
-      [self reCAPTCHAURLWithCompletion:^(NSURL *_Nullable reCAPTCHAURL,
-                                         NSError *_Nullable error) {
+      NSMutableString *eventID = [NSMutableString new];
+      NSUInteger idx = 10;
+      while (--idx >= 0) {
```

Comment:
An `NSUInteger` is always greater than or equal to 0. Just use `int`. And just use a regular for loop, i.e.,

for (int i = 0; i < 10; i++) {
  //...
}